React/Libraries.md

```diff
@@ -0,0 +1,53 @@
+ - [create-react-app](https://github.com/facebook/create-react-app) - Project setup by running one command
+ - [storybook](https://github.com/storybookjs/storybook) - Development environment for UI components
+
+##### React Frameworks
+- [next.js](https://github.com/vercel/next.js) - The React Framework
+
+##### React Routing
+- [next-connect](https://github.com/hoangvvo/next-connect) - Minimal router and middleware layer for Next.js
+- [react-router](https://github.com/ReactTraining/react-router) - Declarative routing for React
+
+##### React Styling and Component Libraries
+- [styled-components](https://github.com/styled-components/styled-components) - Visual primitives for the component age
+- [emotion](https://github.com/emotion-js/emotion) - CSS-in-JS library designed for high performance style composition
+- [theme-ui](https://github.com/system-ui/theme-ui) - Library for creating themeable user interfaces based on constraint-based design principles
+- [chakra-ui](https://github.com/chakra-ui/chakra-ui/) - Simple, Modular & Accessible UI Components for your React Applications
+- [ant-design](https://github.com/ant-design/ant-design) - A UI Design Language and React UI library
+- [styled-system](https://github.com/styled-system/styled-system) - Style props for rapid UI development
+
+##### React Components
+- [react-select](https://github.com/JedWatson/react-select) - The Select Component for React.js
+- [react-datepicker](https://github.com/Hacker0x01/react-datepicker/) - A simple and reusable datepicker component for React
+- [react-virtualized](https://github.com/bvaughn/react-virtualized) - React components for efficiently rendering large lists and tabular data
+- [@tippyjs/react](https://github.com/atomiks/tippyjs-react) - The complete tooltip, popover, dropdown, and menu solution for the web, powered by Popper
+- [downshift](https://github.com/downshift-js/downshift) - A set of primitives to build autocomplete, combobox or select dropdown React components
+- [react-collapse](https://github.com/nkbt/react-collapse) - Component-wrapper for collapse animation with react-motion for elements with variable (and dynamic) height
+- [react-markdown](https://github.com/remarkjs/react-markdown) - Markdown component for React
+- [react-slider](https://github.com/zillow/react-slider) - Accessible, CSS agnostic, slider component for React
+- [react-qr-code](https://github.com/rosskhanas/react-qr-code) - A QR code generator for React and React Native
+- [react-modal](https://github.com/reactjs/react-modal) - Accessible modal dialog component for React
+- [react-cool-inview](https://github.com/wellyshen/react-cool-inview) - React hook to monitor an element enters or leaves the viewport (or another element)
+- [react-avatar-editor](https://github.com/mosch/react-avatar-editor) - Small avatar & profile picture component that allows resizing and cropping uploaded images using a intuitive user interface
+
+##### React Testing
```

Comment:
For react testing we have a lot more libs... take a look at this list I made https://github.com/infinum/JS-React-Example/blob/chakra/docs/testing.md

Reply:
Added missing ones.
Do you mean `@emotion/jest`? `jest-emotion` no longer exists it seems.

Reply:
Emotion 11 got released a few days ago... so they put everything under the `@emotion` namespace... 
Yes!